.github/workflows/ci.yml

```
@@ -10,67 +10,29 @@ jobs:
-        POSTGRES_PASSWORD: test
-        POSTGRES_DB: test_db
-        SECRET_KEY: test-secret-key-for-ci
-        JWT_SECRET_KEY: test-jwt-secret-for-ci
+    - name: Test with Gradle
       run: |
-        pytest --cov=app --cov-report=xml --cov-report=term-missing
+        cd spring-boot-app
+        ./gradlew test --info
     
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v3
```

actionlint (1.7.9)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)